aws/cl_zcash/software/runtime/Readme.md

```diff
@@ -0,0 +1,6 @@
+1. ecdsa_test.cpp use few hardcoded TV for ecdsa signature verify;
+   To compile the ecdsa_test.cpp:
+	make -f makefile_ecdsa
+2. openssl_verify.cpp utilize the openssl lib to generate the test vector dynamiclly
+   To compile the openssl_verify.cpp
+	make -f makefile_ecdsa
```

Comment:
Is this correct?

Reply:
NO some typo there, I am going to correct and make a better version.